### ISSUE TITLE
crash when setting movement mode to left hand

### ISSUE THREAD

**Issue by Tokusei** [mordentral/VRExpPluginExample#5]
Hi, I am getting the following exception when changing `bRightHandMovement` to false in the default properties of VivePawnCharacter blueprint.  Seems like it is trying to setup the controller movement moves and has a reference to a non-existent variable in GraspingHand_Left

`Assertion failed: !Export.ClassIndex.IsNull() [File:D:/Build/++UE4/Sync/Engine/Source/Runtime/CoreUObject/Private/UObject/SavePackage.cpp] [Line: 3789] Export SKEL_GraspingHand_Left_C /Game/VRExpansion/Vive/Testing/GraspingHand/GraspingHand_Left.Default__SKEL_GraspingHand_Left_C class is not mapped when saving /Game/VRExpansion/Vive/Testing/GraspingHand/GraspingHand_Left`

**Comment by Tokusei** [comment]
this crash accours when packaging and deploying for android ARM7

**Comment by mordentral** [comment]
Can you try fixing up redirects on the grasping hand folder? That is an asset ptr issue.

**Comment by Tokusei** [comment]
Hi, I have fixed redirectors in the folder, However the crash still occurs.

**Comment by mordentral** [comment]
Can you open up the skeletal mesh for the left hand and the left hand itself and save and compile and save them?

If this fails you may have to re-create the skeleton, its the engine failing to find a directory path.

**Comment by Tokusei** [comment]
Hi, If I open and recompile these assets, The editor will start crashing, This is even if I dont change the boolean flag in the pawn class to use right handed movement. 
Looks like its trying to save `SKEL_GraspingHand_Left` which doesnt exist, The only skeleton asset in the folder is `SKEL_UE4Mann_LeftHand` I tried creating a copy of it and renaming, however it still seems to be crashing
Is there a hard reference to this asset in the plugin code or something?

**Comment by Tokusei** [comment]
Hi, I found a fix, I renamed the skeleton to `SKEL_GraspingHand_Left ` then duplicated it. I retargeted the original to the duplicate skeleton and it seems to be building. 
Renaming/Replacing References didnt seem to work. Not sure what causes the issue but hopefully it will be stable when packaging 
![image](https://user-images.githubusercontent.com/4631534/108511582-133c2a80-72fb-11eb-92fb-45f0b66dd83b.png)
